0976_Largest_Perimeter_Triangle/try_1.cpp

```diff
@@ -0,0 +1,12 @@
+class Solution {
+public:
+    int largestPerimeter(vector<int>& nums) {
+        sort(nums.begin(), nums.end());

```

Comment:
<只是提醒>也可以使用這樣的方式從大到小排列
```suggestion
        sort(nums.rbegin(), nums.rend());
```

Reply:
這是decending order對吧?
https://www.geeksforgeeks.org/vector-rbegin-and-rend-function-in-c-stl/

Reply:
對的 
C++的STL container我都是到官方document看的
像是vector -> https://cplusplus.com/reference/vector/vector/?kw=vector
裡面會把所有member function都列出來